docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -721,6 +721,53 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[truncateIntervalHour]
 include-tagged::{sql-specs}/docs/docs.csv-spec[truncateIntervalDay]
 --------------------------------------------------
 
+[[sql-functions-datetime-format]]
+==== `FORMAT`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+FORMAT(
+    date_exp/datetime_exp/time_exp, <1>
+    string_exp) <2>
+--------------------------------------------------
+
+*Input*:
+
+<1> date/datetime/time expression
+<2> format pattern
+
+*Output*: string
+
+*Description*: Returns the date/datetime/time as a string using the format specified in the 2nd argument. The formatting
+pattern used is the one from
+https://docs.microsoft.com/en-us/sql/t-sql/functions/format-transact-sql?view=sql-server-ver15#ExampleD[Microsoft SQL Server Format Specification].
```

Comment:
```suggestion
https://docs.microsoft.com/en-us/sql/t-sql/functions/format-transact-sql#ExampleD[Microsoft SQL Server Format Specification].
```

`view` URL param's superfluous, but not hurting either.

However, wouldn't the [source reference page](https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings) itself be more suitable as the format reference, instead of an example page? We could/should still reference the SQL Server `FORMAT` function (in the lines of "as used by the [`FORMAT`](https://docs.microsoft.com/en-us/sql/t-sql/functions/format-transact-sql#argumentsl) function" or similar) to link it back to SQL, however.

Reply:
Yep, I agree!

Reply:
updated documentation